### PR TITLE
docs: change-parameters proposal

### PR DESCRIPTION
**Pull request by ptrus**

### CODE REVIEW

File: docs/network.md [hunk 1]
```diff
@@ -206,6 +206,8 @@ The following proposal types are currently supported:
 
 - `upgrade <descriptor.json>`: Network upgrade proposal. Provide a JSON file
   containing the upgrade descriptor.
+- `parameter-change <module-name> <changes.json>`: Network parameter change
+  proposal. Provide the module name and the consensus parameter changes JSON.
```
Comment (matevz) [inline docs/network.md:210]: Is there a place where all possible module names are listed?

File: docs/network.md [hunk 2]
```diff
@@ -206,6 +206,8 @@ The following proposal types are currently supported:
 
 - `upgrade <descriptor.json>`: Network upgrade proposal. Provide a JSON file
   containing the upgrade descriptor.
+- `parameter-change <module-name> <changes.json>`: Network parameter change
```
Comment (matevz) [inline docs/network.md:209]: Let's order the proposal names by name (cancel-upgrade, parameter-change, upgrade).

File: docs/network.md [hunk 3]
```diff
@@ -319,7 +321,7 @@ The provided ID can be one of the following:
 
 - If the [ParaTime ID] is provided, Oasis CLI shows ParaTime information stored
   in the network's registry.
-  
+
```
Comment (matevz) [inline docs/network.md:324]: Revert.

File: docs/network.md [hunk 4]
```diff
@@ -345,7 +347,7 @@ The provided ID can be one of the following:
   ParaTimes, trusted execution environment support and more. For example:
 
   ![code shell](../examples/network-show/id-node.in.static)
-  
+
```
Comment (matevz) [inline docs/network.md:350]: Revert.

**Review by matevz — CHANGES_REQUESTED** [review]

File: docs/network.md (hunk 1 above)
Comment (ptrus) [reply docs/network.md:210]: These are the consensus modules. Not sure if there's a list somewhere. Idk how useful even would be listing that, someone creating a parameter change proposal needs somewhat in-depth knowledge of the oasis-core code anyway.

**Pull request merged**